pkg/operator/deploy/staticresources/master/deployment.yaml

```diff
@@ -30,6 +30,12 @@ spec:
         ports:
         - containerPort: 8080
           name: http
+        livenessProbe:
+          httpGet:
+            path: /healthz/ready
+            port: 8080
+          initialDelay: 5
+          periodSeconds: 5
```

Comment:
I think we can leave these at the defaults - i.e. don't specify them (ditto worker)